go/tools/bzltestutil/lcov.go

```diff
@@ -173,16 +182,10 @@ type LcovTestDeps struct {
 //
 // This gives us a good place to intercept the os.Exit(m.Run()) with coverage
 // data already available.
-func (ltd LcovTestDeps) SetPanicOnExit0(panicOnExit bool) {
-	if !panicOnExit {
-		lcovAtExitHook()
-	}
-	ltd.TestDeps.SetPanicOnExit0(ltd.OriginalPanicOnExit)
-}
-
-func lcovAtExitHook() {
-	if err := ConvertCoverToLcov(); err != nil {
-		log.Printf("Failed to collect coverage: %s", err)
-		os.Exit(TestWrapperAbnormalExit)
-	}
-}
+//
+// Note: This method is deprecated and no longer used now that go's coverageredesign
```

Comment:
If it's truly no longer used we can drop it, there are no compatibility guarantees for the internal package.

Reply:
removed!